sw/airborne/subsystems/abi_sender_ids.h

```diff
@@ -278,6 +278,13 @@
 #endif
 
 /*
+ * IDs of VELOCITY estimates (message 12)
+ */
+#ifndef VEL_DRAGSPEED_ID
+#define VEL_DRAGSPEED_ID 1
```

Comment:
I was quite surprised about this but then I realized that the other velocity ids where in the wrong place. 

Note to self. We need to update this for the opticflow_module, stereocamera and the px4flow which are all wrong.